R/Rules-class.R

```diff
@@ -297,6 +297,28 @@ validObject(NextBestMinDist(0.1))
 
 ## ============================================================
 
+#' Class for Next Best Dose based on Information Theory.
+#'
+#'
+#' @slot target target DLT probability.
+#' @slot asymmetry value of the asymmetry exponent
+#'
+#' @export
+#' @aliases NextBestInfTh
+.NextBestInfTh <- setClass(Class = "NextBestInfTh",
+                           contains = "NextBest",
+                           representation(target = "numeric", asymmetry="numeric"))
+validObject(.NextBestInfTh())
```

Comment:
I think we no longer put this `validObject()` after the class definitions now, so you can omit these